src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector(`.logo`);
+
+function createNotification(bool) {
+  if (bool) {
+    const message = document.createElement('div');
+
+    message.classList.add('message');
+    message.innerText = 'Promise was resolved!';
+    document.body.append(message);
+  } else {
+    const errorMessage = document.createElement('div');
+
+    errorMessage.classList.add('error-message');
+    errorMessage.innerText = 'Promise was rejected!';
+    document.body.append(errorMessage);
+  }
+}
+
+const firstPromise = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve(true);
```

Comment:
```suggestion
    resolve('Promise was resolved');
```
Передавай сообщение тут сразу, тогда ты сразу сможешь использовать его в функции createNotification